manifests/exploded_war.pp

```diff
@@ -37,6 +37,12 @@
     content => template('i2b2/log4j.properties.erb'),
   }
 
+  exec { 'replace-password' :
+    cwd         => "$dir/WEB-INF/conf",
+    command     => 'sed -i \'s#<parameter name="password">axis2</parameter>#<parameter name="password">Z61OaOvXAfsyIJrQ</parameter>#\' axis2.xml',
+    refreshonly => true,
+  }
+
```

Comment:
* Well, I was not sure about this but I wanted to write 'requires => extract $axis_war' but I thought this was implied by the creates and cwd.
* The point is that when some entity encounters this service and happens to find axis2, it will try the default password first. We have to do something about this anyway because the other passwords are also readable on github. We just need to replace them with a password that we get elsewhere. 
* Not just listing, also removing and thereby breaking the service.